pages/account/[[...index]].tsx

```diff
@@ -231,14 +240,28 @@ export async function getServerSideProps({ req }) {
       },
     }
   }
+  // Parsing path from url to pass to ProfileTabs.
+  const tabsPath = req.url.split("/", -1)[2] || null
```

Comment:
You can do this now or create a ticket for this:

This works for any string that is not part of the tabs because in `ProfileTabs` the default is the first tab -- this is good. But it can be confusing to have a working site and have a bad url, for example `/account/shouldnotwork`:

![Screen Shot 2024-03-04 at 1 54 15 PM](https://github.com/NYPL/research-catalog/assets/1280564/b5dd4864-ec93-4062-8d2b-69589ad36dba)

So I think you can redirect here, similar to the code you have in 249-256, if `tabsPath` is not in one of the expected urls. This could be if there's typo in the url. I don't expect this to be a big issue since users will be using the tabs and not typing the url directly, but _if_ they do we want to make sure we cover these cases.

Reply:
Ticket is created and pulled into this sprint! https://jira.nypl.org/browse/SCC-4024 I'd like to make this a hook that handles all /account redirects, similar to the useLogOutRedirect, so we can follow the same testing pattern